Update CLARIN-IS-recommendation.xml

starkadur opened the pull request.

starkadur commented:
I have added reccomended formats for CLARIN-IS

bansp commented:
Thank you and apologies for taking a while with that.

bansp commented:
I have fixed some typos; you can see the result in https://github.com/clarin-eric/standards/commit/1a920f9c45958e9cf0c6a08283938d2d15175518 .

One remark: you might want to add your centre's description to the `<info>` element. One robust example of that is the EL centre, but if you look at others, it is also possible to add just a few sentences to make the whole thing look better and maintained.

Speaking of maintenance: the recommendations will not be listed as "curated" until the `<respStmt>` element is filled. What I could do for you is:
```xml
        <respStmt>
            <name>Starkaður Barkarson</name>
            <link>https://github.com/starkadur</link>
            <reviewDate>2025-03-21</reviewDate>
        </respStmt>
```
-- if you give me green light for that.

starkadur commented:
Hi Piotr.

I added respStmt and info and created a pull request.

Regards,
 Starkaður
________________________________
From: Piotr Banski ***@***.***>
Sent: Friday, March 21, 2025 10:57 AM
To: clarin-eric/standards ***@***.***>
Cc: Starkaður Barkarson - AST ***@***.***>; Author ***@***.***>
Subject: Re: [clarin-eric/standards] Update CLARIN-IS-recommendation.xml (PR #376)


I have fixed some typos; you can see the result in 1a920f9<https://github.com/clarin-eric/standards/commit/1a920f9c45958e9cf0c6a08283938d2d15175518> .

One remark: you might want to add your centre's description to the <info> element. One robust example of that is the EL centre, but if you look at others, it is also possible to add just a few sentences to make the whole thing look better and maintained.

Speaking of maintenance: the recommendations will not be listed as "curated" until the <respStmt> element is filled. What I could do for you is:

        <respStmt>
            <name>Starkaður Barkarson</name>
            <link>https://github.com/starkadur</link>
            <reviewDate>2025-03-21</reviewDate>
        </respStmt>

-- if you give me green light for that.

—
Reply to this email directly, view it on GitHub<https://github.com/clarin-eric/standards/pull/376#issuecomment-2743017283>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AD3OIDT22YXMXLVQ3XS7PNL2VPWADAVCNFSM6AAAAABYWRIP62VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONBTGAYTOMRYGM>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

[bansp]bansp left a comment (clarin-eric/standards#376)<https://github.com/clarin-eric/standards/pull/376#issuecomment-2743017283>

I have fixed some typos; you can see the result in 1a920f9<https://github.com/clarin-eric/standards/commit/1a920f9c45958e9cf0c6a08283938d2d15175518> .

One remark: you might want to add your centre's description to the <info> element. One robust example of that is the EL centre, but if you look at others, it is also possible to add just a few sentences to make the whole thing look better and maintained.

Speaking of maintenance: the recommendations will not be listed as "curated" until the <respStmt> element is filled. What I could do for you is:

        <respStmt>
            <name>Starkaður Barkarson</name>
            <link>https://github.com/starkadur</link>
            <reviewDate>2025-03-21</reviewDate>
        </respStmt>

-- if you give me green light for that.

—
Reply to this email directly, view it on GitHub<https://github.com/clarin-eric/standards/pull/376#issuecomment-2743017283>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AD3OIDT22YXMXLVQ3XS7PNL2VPWADAVCNFSM6AAAAABYWRIP62VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONBTGAYTOMRYGM>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
